pkgs/by-name/wg/wget/package.nix

```diff
@@ -18,18 +36,36 @@ stdenv.mkDerivation rec {
     patchShebangs doc
   '';
 
-  nativeBuildInputs = [ gettext pkg-config perlPackages.perl lzip libiconv libintl nukeReferences ];
-  buildInputs = [ libidn2 zlib pcre libuuid ]
+  nativeBuildInputs = [
+    gettext
+    pkg-config
+    perlPackages.perl
```

Comment:
```suggestion
    perl
```

Reply:
`perlPackages` is referenced below in checkInputs